### PR TITLE
README, CONTRIBUTING, and godoc updates.

### PR DESCRIPTION
**Pull request by TristonianJones**
#1 README.md updated with simple example and instructions how to run tests.
#2 CONTRIBUTING.md states guidelines for contributions and additional references.

Godoc comments have been updated to be more consistent with Go documentation
style guides.

### CODE REVIEW

File: PULL_REQUEST_TEMPLATE.md
Comment (rachelmyers) [inline PULL_REQUEST_TEMPLATE.md]: I would change this to `After a CEL maintainer has approved the change, it may be merged by the author if the author has write access. Otherwise it will be merged by a maintainer.`

File: PULL_REQUEST_TEMPLATE.md
Comment (rachelmyers) [inline PULL_REQUEST_TEMPLATE.md]: TL;DR: I would change this to `Multiple commits in a PR should be squashed before merging`. 

Practically, it's easier to either ask people to squash all commits in a PR together, or let them merge their potentially messy history. Squashing some but not all is more complicated than one `squash` command, and now there's an option in dropdown next to the merge button to squash all commits.

File: README.md
Comment (rachelmyers) [inline README.md]: It looks a little counter intuitive to not have a link without an image. What would you think of:
```See [Go doc](https://godoc.org/github.com/google/cel-go) to learn how to integrate CEL into services written in Go. [GoDoc](https://godoc.org/github.com/google/cel-go)```

**Review by rachelmyers — APPROVED** [review]
This looks great; it's a great idea to include descriptions of CEL itself in the README. The comments I left are about making it easier for people to squash and merge in one step, and a nit about linking to GoDoc. 👍

File: README.md
Comment (TristonianJones) [reply README.md]: Good suggestion.

File: PULL_REQUEST_TEMPLATE.md
Comment (TristonianJones) [reply PULL_REQUEST_TEMPLATE.md]: Sure, happy to update.

File: PULL_REQUEST_TEMPLATE.md
Comment (TristonianJones) [reply PULL_REQUEST_TEMPLATE.md]: Sounds good. I've made a minor edit to say 'If a CEL maintainer' rather than 'After' or 'Once' as there is no guarantee of approval, but kept the rest of the suggested phrasing.

**Pull request merged**
